config/locales/zxcvbn/fr.yml

```diff
@@ -0,0 +1 @@
+--- {}
```

Comment:
Yes, it is. It looks like it didn't get properly populated. @zachmargolis Is this the parsing issue you were mentioning?